caluma_client/models.py

```diff
@@ -0,0 +1,179 @@
+import base64
+
+from .parser import parse_document
+
+ANSWER_TYPE_MAP = {
+    "TextQuestion": "StringAnswer",
+    "TextareaQuestion": "StringAnswer",
+    "IntegerQuestion": "IntegerAnswer",
+    "FloatQuestion": "FloatAnswer",
+    "MultipleChoiceQuestion": "ListAnswer",
+    "ChoiceQuestion": "StringAnswer",
+    "DynamicMultipleChoiceQuestion": "ListAnswer",
+    "DynamicChoiceQuestion": "StringAnswer",
+    "TableQuestion": "TableAnswer",
+    "FormQuestion": "FormAnswer",
```

Comment:
There is no `FormAnswer`, this should be `None` as well.

Reply:
Hmm ok this is adapted from https://github.com/projectcaluma/ember-caluma/blob/master/addon/lib/field.js#L41
Maybe I should suggest this in ember-caluma as well.

Reply:
That would be great!

There was a `FormAnswer` once, but it broke everything (nested Answers). So we removed it.

Reply:
Done